src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColumnBtn = document.querySelector('.append-column');
+const removeColumnBtn = document.querySelector('.remove-column');
+
+appendRowBtn.addEventListener('click', () => {
+  const rowCount = table.rows.length;
+
+  if (rowCount === 9) {
+    appendRowBtn.disabled = true;
```

Comment:
This line disables the appendRowBtn when the row count reaches 9, but it doesn't re-enable it if rows are removed later. Consider adding logic to re-enable the button when the row count decreases below 9.